### DISCUSSION TITLE
react-sortable-tree alternatives

### DISCUSSION THREAD

**Discussion by fritz-c** [frontend-collective/react-sortable-tree#942] [Announcements]
Thank you for choosing or looking into react-sortable-tree.
Maintenance on this project is indefinitely paused until the day some project at my work needs it updated. I've used personal time to maintain it in the past but do not intend to do that moving forward. I've considered finding different maintainers for the project, but that too would involve a large time investment and follow-up to make sure the library heads in a positive direction in line with the goals of the current implementation.

I welcome competing libraries or well-established, better-maintained forks that fix the existing issues of the current implementation. Please share your recommended alternatives in the discussion here.

**Comment by dx314** [comment]
https://github.com/nosferatu500/react-sortable-tree appears to be the most actively maintained fork of `react-sortable-tree`

**Comment by tianzhich** [comment]
> https://github.com/nosferatu500/react-sortable-tree appears to be the most actively maintained fork of `react-sortable-tree`

Yes, this is a well maintained fork which i also used in my production environment last year. I also fixed some bugs with @nosferatu500. I would like to rewrite the codebases to hooks if i have any time.

**Reply by nosferatu500** [reply]
>  I would like to rewrite the codebases to hooks if i have any time

I have some plans for this. But I'm not working on the project I forked this for, but I still want to make some changes if I have time.

1. Hooks based.
2. Multi selection support.
3. Better themes support.
4. Maybe resolve some performance issues but not sure.

I just need time for this.

**Reply by tianzhich** [reply]
Hey @nosferatu500 I just leaved my last company and i'll have time in the next 3 months. So maybe I could do some works 😎

**Comment by peymanahmadifar** [comment]
https://github.com/lukasbach/react-complex-tree
